tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "declaration": true,
     "outDir": "./dist",
     "strict": true,
-    "typeRoots": ["node_modules/@types"]
+    "typeRoots": ["node_modules/@types", "customTypes"]
```

Comment:
Is this still required?

Reply:
@paddybasi - oh no, I forgot that too! :)